examples/coreos/units-etcd/ceph-mds@.service

```diff
@@ -0,0 +1,27 @@
+[Unit]
+Description=Ceph Metadata Service
+After=docker.service
+
+[Service]
+EnvironmentFile=/etc/environment
+; Note: the MDS_NAME must not be (solely?) numeric.  I use "a", "b", etc (ceph-mds@a, ceph-mds@b, etc)
+Environment=MDS_NAME=%i
+ExecStartPre=-/usr/bin/docker kill %p
+ExecStartPre=-/usr/bin/docker rm %p
+ExecStart=/usr/bin/docker run --rm --name %p --net=host \
```

Comment:
Same above. But I copied this from yours: https://github.com/ceph/ceph-docker/blob/master/examples/coreos/units/ceph-mds@.service#L11 :)